docs/solidity-by-example.rst

```diff
@@ -558,26 +545,26 @@ Safe Remote Purchase
         function Purchase() payable {
             seller = msg.sender;
             value = msg.value / 2;
-            if (2 * value != msg.value) throw;
+            require(2 * value == msg.value);
```

Comment:
I actually don't think so. it is a simple mathematical equation, nobody would add parentheses there.